tests/python-pytest/onnx/onnx_test.py

```diff
@@ -0,0 +1,133 @@
+    input1 = np.random.ranf([3, 10]).astype("float32")
+    output = mxnet_backend.run_node(node_def, [input1])[0]
+    numpy_op = np.sum(input1, axis=(1, 0), keepdims=True)
+    npt.assert_almost_equal(output, numpy_op, decimal=5)
+
+@with_seed()
+def test_reduce_prod():
+    """Test for ReduceProd operator"""
+    node_def = helper.make_node("ReduceProd", ["input1"], ["output"], axes=[1, 0], keepdims=1)
+    input1 = np.random.ranf([3, 10]).astype("float32")
+    output = mxnet_backend.run_node(node_def, [input1])[0]
+    numpy_op = np.prod(input1, axis=(1, 0), keepdims=True)
+    npt.assert_almost_equal(output, numpy_op, decimal=5)
+
+@with_seed()
+def test_squeeze():
+    """Test for Squeeze operator"""
+    node_def = helper.make_node("Squeeze", ["input1"], ["output"], axes=[1, 3])
+    input1 = np.random.ranf([3, 1, 2, 1, 4]).astype("float32")
+    output = mxnet_backend.run_node(node_def, [input1])[0]
+    npt.assert_almost_equal(output, np.squeeze(input1, axis=[1, 3]))
+
+def test_super_resolution_example():
+    """Test the super resolution example in the example/onnx folder"""
+    sys.path.insert(0, os.path.join(CURR_PATH, '../../../example/onnx/'))
+    import super_resolution
+
+    sym, params = super_resolution.import_onnx()
+    assert sym is not None
+    assert params is not None
+
+    inputs = sym.list_inputs()
+    assert len(inputs) == 9
+    for i, input_param in enumerate(['param_7', 'param_5', 'param_3', 'param_1',
+                                     'input_0', 'param_0', 'param_2', 'param_4', 'param_6']):
+        assert inputs[i] == input_param
+
+    assert len(sym.list_outputs()) == 1
+    assert sym.list_outputs()[0] == 'reshape5_output'
+
+    attrs_keys = sym.attr_dict().keys()
+    assert len(attrs_keys) == 19
+    for i, key_item in enumerate(['reshape4', 'param_5', 'param_4', 'param_7',
+                                  'param_6', 'param_1', 'param_0', 'param_3',
+                                  'param_2', 'reshape2', 'reshape3', 'reshape0',
+                                  'reshape1', 'convolution2', 'convolution3',
+                                  'convolution0', 'convolution1', 'reshape5',
+                                  'transpose0']):
+        assert key_item in attrs_keys
+
+    param_keys = params.keys()
+    assert len(param_keys) == 8
+    for i, param_item in enumerate(['param_5', 'param_4', 'param_7', 'param_6',
+                                    'param_1', 'param_0', 'param_3', 'param_2']):
+        assert param_item in param_keys
+
+    logging.getLogger().info("Asserted the result of the onnx model conversion")
+
+    output_img_dim = 672
+    input_image, img_cb, img_cr = super_resolution.get_test_image()
+    result_img = super_resolution.perform_inference(sym, params, input_image,
+                                                    img_cb, img_cr)
+    assert result_img.size == (output_img_dim, output_img_dim)
```

Comment:
Please add a correctness test. Hashing the output and comparing that could be a good way.